src/GraphQL/GlobalSwitches.cs

```diff
@@ -65,6 +65,13 @@ public static class GlobalSwitches
     /// </summary>
     public static bool UseDeclaringTypeNames { get; set; } = false;
 
+    /// <summary>
+    /// Enable this switch to see additional debugging information in exception messages during schema initialization.
+    /// <br/>
+    /// By default disabled.
+    /// </summary>
+    public static bool TrackGraphTypeInitialization { get; set; } = true;
```

Comment:
```suggestion
    public static bool TrackGraphTypeInitialization { get; set; } = false;
```

Reply:
I don't think all this extra trace activity and metadata logging should be enabled by default.

Reply:
I forgot to remote it 🤦‍♂️ .